pages/1.10/release-notes/1.10.5/index.md

```diff
@@ -0,0 +1,251 @@
+---
+layout: layout.pug
+navigationTitle:  Release Notes for 1.10.5
+title: Release Notes for 1.10.5
+menuWeight: 0
+excerpt:
+---
+
+These are the release notes for DC/OS 1.10.5.
+
+[button color="purple" href="https://downloads.dcos.io/dcos/stable/1.10.5/dcos_generate_config.sh"]Download DC/OS Open Source[/button]
+
+
+# Issues Fixed in DC/OS 1.10.5
+
+- COPS-1963 - The agent admin router no longer exposes the default Nginx page.
+- DCOS-13824 - Removed list of cluster packages from cloud config and turned it into a build artifact that is downloaded during node setup.
+- DCOS-18784 - The DC/OS UI now shows the correct number of instances in a pod.
+- DCOS-19008 - Master certificates now include exhibitor_address if exhibitor_address is an IP address. [enterprise type="inline" size="small"/]
+- DCOS-20629 - Fixed a deadlock in the go-zookeeper library.
+- DCOS-20683 - Fixed an error in encoding metrics to JSON when executing the DataDog plugin for DC/OS.
+- DCOS-20158 - Updated marked npm package to address two security vulnerabilities: [CVE-2017-17461](https://nvd.nist.gov/vuln/detail/CVE-2017-17461) and [CVE-2017-1000427](https://nvd.nist.gov/vuln/detail/CVE-2017-1000427).
+- DCOS-20184 - Users can now be authenticated if two non-leading master nodes are simultaneously partitioned from the network, via IP tables.
+- DCOS-20352 - Fixed a configuration issue with CockroachDB related to the number of master nodes.
```

Comment:
Do we have a JIRA related to upgrading Cockroach to 1.1.4?  I think this is https://jira.mesosphere.com/browse/DCOS-19694